### PR TITLE
Fix dispatch_str for MoveFocusedWindowToWorkspace(Silent)

### PR DESCRIPTION
**Pull request by sij1nk**
This MR fixes the previously incorrect dispatch string for `MoveFocusedWindowToWorkspace(Silent)`. Since we want to move the window, we want `movetoworkspace(silent)` instead of `workspace` here

### CODE REVIEW

**Comment by yavko** [comment]
This is intended, it's Move**Focused**WindowToWorkspace, MoveToWorkspace exists for what you want

**Comment by sij1nk** [comment]
Sorry but I think you misunderstood

Currently `MoveFocusedWindowToWorkspace` and `MoveFocusedWindowToWorkspaceSilent` use the `workspace` dispatcher, which doesn't actually move the focused window - it just changes the workspace

**Comment by yavko** [comment]
> Sorry but I think you misunderstood
> 
> Currently `MoveFocusedWindowToWorkspace` and `MoveFocusedWindowToWorkspaceSilent` use the `workspace` dispatcher, which doesn't actually move the focused window - it just changes the workspace

You're right, but if I'm reading this right you aren't doing it that way if that's the case, you have to get rid of the parameter and the `{sep}{work}` in the format string.

**Comment by yavko** [comment]
> > Sorry but I think you misunderstood
> > Currently `MoveFocusedWindowToWorkspace` and `MoveFocusedWindowToWorkspaceSilent` use the `workspace` dispatcher, which doesn't actually move the focused window - it just changes the workspace
> 
> You're right, but if I'm reading this right you aren't doing it that way if that's the case, you have to get rid of the parameter and the `{sep}{work}` in the format string.

nvm i read it wrong

File: src/dispatch.rs [hunk 1]
```diff
@@ -480,8 +480,8 @@ pub(crate) fn gen_dispatch_str(cmd: DispatchType, dispatch: bool) -> crate::Resu
         MoveToWorkspace(work, None) => format!("movetoworkspace{sep}{work}"),
         MoveToWorkspaceSilent(work, Some(win)) => format!("movetoworkspacesilent{sep}{work},{win}"),
         MoveToWorkspaceSilent(work, None) => format!("movetoworkspacesilent{sep}{work}"),
-        MoveFocusedWindowToWorkspace(work) => format!("workspace{sep}{work}"),
-        MoveFocusedWindowToWorkspaceSilent(work) => format!("workspace{sep}{work}"),
+        MoveFocusedWindowToWorkspace(work) => format!("movetoworkspace{sep}{work}"),
```
Comment (yavko) [inline src/dispatch.rs:483]: Instead, maybe we can deprecate this one and keep only `MoveToWorkspace` as it already uses an option for this

**Review by yavko — DISMISSED** [review]
LGTM

File: src/dispatch.rs (hunk 1 above)
Comment (sij1nk) [reply src/dispatch.rs]: done (I hope this is what you meant)

Comment (yavko) [reply src/dispatch.rs]: No use the `#[deprecated]` attribute (also add a reason for deprecation), so it won't break API, but I'll remove it in `0.4` or `1.0`.

Comment (sij1nk) [reply src/dispatch.rs:488]: ah I see, thanks. Deprecated them instead

Cargo now complains about the use of the deprecated field within the lib, when compiling the lib itself; should I silence the warning or leave it as it is?

**Review by yavko — APPROVED** [review]

**Pull request merged**